packages/client/hmi-client/src/components/models/Model.vue

```diff
@@ -3,8 +3,17 @@
 		<header>
 			<div class="framework">{{ model?.framework }}</div>
 			<div class="header-and-buttons">
-				<h4 v-html="title" />
-				<span v-if="isEditable">
+				<InputText
+					v-if="assetId === ''"
```

Comment:
You could simplify to `v-if="assetId"`

Reply:
yeah, I was toying around with another falsy state so I kept this explicit.  Didn't remove it.